mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -157,6 +157,13 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 							Required:    true,
 							Description: `The machine type to use. See tiers for more details and supported versions. Postgres supports only shared-core machine types, and custom machine types such as db-custom-2-13312. See the Custom Machine Type Documentation to learn about specifying custom machine types.`,
 						},
+						"edition": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "ENTERPRISE",
```

Comment:
I wonder if the default value is needed here. The default value breaks other tests.

Also, I do not see the doc for the new fields in the API doc. https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1/instances#Settings. Where I can find the doc for the new fields?

Reply:
Documentation is not updated yet as launch is scheduled for 12/7, but you can find it definition [here](https://github.com/googleapis/google-api-go-client/blob/24b4d0b3c502e1f3cd7d73fdfcf039fe8b6fa08c/sqladmin/v1beta4/sqladmin-gen.go#L3910C6-L3910C6).

I have removed the 'defaults to' as it is not necessary and now won't cause any issue with other tests.